packages/components/action-link/README.md

```diff
@@ -12,7 +12,8 @@ To discuss or contribute to this component, visit the [GitHub issue for this com
 
 ```html
 <div class="nhsuk-action-link">
-  <a class="nhsuk-action-link__link" href="https://www.nhs.uk/service-search/minor-injuries-unit/locationsearch/551">
+  <a class="nhsuk-action-link__link"
```

Comment:
Despite the HTML outputting like this from the nunjucks macro can we put this all on 1 line and not across 2?